c/docs/answer/explanation.adoc

```diff
@@ -207,23 +221,6 @@ A string representation of this ``Explainables`` object
 .Returns
 `char*`
 
-[#_methods__answer__explanation]
-=== explanation
-
-[#_Struct_Explanation]
-==== Struct Explanation
-
-
-
-An explanation of which rule was used for inferring the explained concept, the condition of the rule, the conclusion of the rule, and the mapping of variables between the query and the rule’s conclusion.
-
-[#_Struct_ExplanationIterator]
-==== Struct ExplanationIterator
-
-
-
-Iterator over the ``Explanation``s in the result of the explain query.
-
```

Comment:
The change re-ordered some of these. I believe the structs are grouped together on top now.